homeassistant/components/skybell/light.py

```diff
@@ -31,16 +30,15 @@ async def async_setup_entry(
 class SkybellLight(SkybellEntity, LightEntity):
     """A light implementation for Skybell devices."""
 
-    _attr_supported_color_modes = {ColorMode.BRIGHTNESS, ColorMode.RGB}
+    _attr_color_mode = ColorMode.RGB
+    _attr_supported_color_modes = {ColorMode.RGB}
 
     async def async_turn_on(self, **kwargs: Any) -> None:
         """Turn on the light."""
-        if ATTR_RGB_COLOR in kwargs:
-            rgb = kwargs[ATTR_RGB_COLOR]
-            await self._device.async_set_setting(ATTR_RGB_COLOR, rgb)
-        if ATTR_BRIGHTNESS in kwargs:
-            level = int((kwargs.get(ATTR_BRIGHTNESS, 0) * 100) / 255)
-            await self._device.async_set_setting(ATTR_BRIGHTNESS, level)
+        for key, value in kwargs.items():
+            if key == ATTR_BRIGHTNESS:
+                value = int(value * 100 / 255)
+            await self._device.async_set_setting(key, value)
```

Comment:
It is a bit strange that the library takes Home Assistant constants.  There really should be a translation layer between Home Assistant constants and skybell setting names since the Home Assistant ones may change which would break skybell

Reply:
Possible but is it probable that would happen?

Reply:
We already changed all the color modes once so I'd say yes